examples/manual-p2p/src/main.rs

```diff
@@ -0,0 +1,165 @@
+//! Low level example of connecting to and communicating with a peer.
+//!
+//! Run with
+//!
+//! ```not_rust
+//! cargo run -p manual-p2p
+//! ```
+
+use std::time::Duration;
+
+use colored::*;
```

Comment:
I think we'll do just fine with regular prints, the messages are clear enough.

please use regular printlns

Reply:
good point, the colored prints have been removed